tests/unit_tests/explainer/test_smart_plotter.py

```diff
@@ -2024,3 +2025,36 @@ def test_correlations_2(self):
         assert len(output.data[0].x) == 3
         assert len(output.data[0].y) == 3
         assert output.data[0].z.shape == (3, 3)
+
+    def test_stability_plot(self):
+        df = pd.DataFrame(np.random.randint(0, 100, size=(15, 4)), columns=list('ABCD'))
+        X = df.iloc[:, :-1]
+        y = df.iloc[:, -1]
+        model = LinearRegression().fit(X, y)
+
+        xpl = SmartExplainer()
+        xpl.compile(x=X,
+                    model=model,
+                    y_pred=y
+                    )
+
+        output = xpl.plot.stability_plot(distribution="none")
+
+        assert len(output.data[0]["x"]) == X.shape[1]
+        assert len(output.data[0]["y"]) == X.shape[1]
+
```

Comment:
you can add test for violin and boxplot
check the type of Data elements (plot.data[0])
check max_feature param
check selection

you should patch compute_features_stability

Reply:
you have to add a test for compute_features_stability and store_features_stability in test_smart_explainer

Reply:
The method `store_features_stability` has been removed, while `compute_features_stability`is already in test_smart_explainer if I remember correctly.

Regarding the patch, I haven't used it so far. As calculations are pretty fast (I use a test data frame with 15 rows and 4 columns), what are the other advantages ? Plus I would have to figure out the correct output formats to make it work

Reply:
the interest of @patch is to allow us to test just the part of the code that interests us.
-> when a test fails, the diagnosis is made more quickly.

we can discuss it, or even improve this part later